zenoh_c_vendor/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(ZENOHC_CARGO_FLAGS "--no-default-features$<SEMICOLON>--features=zenoh/transp
 # Set VCS_VERSION to include latest changes from zenoh-c https://github.com/eclipse-zenoh/zenoh-c/pull/340.
```

Comment:
@JEnoch can we update the comment here to reference the PR associated with the new commit hash?

Reply:
Done!